### PR TITLE
Add readonly check func

### PR DESCRIPTION
**Pull request by kylazhang**
Signed-off-by: Kylazhang <weizhan@redhat.com>

### CODE REVIEW

File: virttest/utils_test/libvirt.py [hunk 1]
```diff
@@ -3550,3 +3550,19 @@ def check_qemu_cmd_line(content, err_ignore=False):
             raise exceptions.TestFail("Expected '%s' was not found in "
                                       "qemu command line" % content)
     return True
+
+
+def readonly_failure_check(cmd_result):
```
Comment (balamuruhans) [inline virttest/utils_test/libvirt.py]: can we have `msg` as default arg,
```python
def readonly_failure_check(cmd_result, msg="forbidden.* read only"):
```

**Comment by balamuruhans** [comment]
I think we can reuse,

https://github.com/avocado-framework/avocado-vt/blob/32f49b1a488a933880c86dd6a9214b77098e6751/virttest/utils_test/libvirt.py#L1514-L1526

File: virttest/utils_test/libvirt.py (hunk 1 above)
Comment (kylazhang) [reply virttest/utils_test/libvirt.py]: Ok, thanks for remind

**Comment by kylazhang** [comment]
@balamuruhans yes, thanks, I will close it. Thanks

**Pull request closed without merging**